.github/workflows/test.yaml

```diff
@@ -123,7 +124,7 @@ jobs:
 
   test-e2e:
     name: e2e
-    if: github.head_ref == 'changeset-release/master'
+    if: startsWith(github.head_ref, 'changeset-release/') == 'true'
```

Comment:
We probably can't run live e2e for patch releases of old versions, as they'll get run against a beta-5 url and will fail.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1849/commits/9a4987ee81db0410ef8916cb598ff856d9c9b261